netty/src/main/java/io/grpc/netty/SendResponseHeadersCommand.java

```diff
@@ -67,19 +67,19 @@ public boolean equals(Object that) {
       return false;
     }
     SendResponseHeadersCommand thatCmd = (SendResponseHeadersCommand) that;
-    return thatCmd.streamId == streamId
+    return thatCmd.stream == stream
         && thatCmd.headers.equals(headers)
         && thatCmd.endOfStream == endOfStream;
   }
 
   @Override
   public String toString() {
-    return getClass().getSimpleName() + "(streamId=" + streamId + ", headers=" + headers
+    return getClass().getSimpleName() + "(stream=" + stream + ", headers=" + headers
```

Comment:
Now using `stream.id()`.

`toStringHelper` sounds fine, but I'd rather not make unnecessary changes in this PR, given it's hard in this change to avoid scope creep.